source/incident-collaboration/running-playbooks.rst

```diff
@@ -100,7 +100,7 @@ If a playbook run was ended prematurely, it can be restarted within the channel
 Playbook dashboard
 ------------------
 
-Each playbook has a dashboard with details such as how many runs using the playbook are currently in progress, total runs per week over a 12-week period, and a list of runs with details such as status, duration, owner/participants, and tasks finished.
+Each playbook has a dashboard with details such as how many runs using the playbook are currently in progress, total runs per week over a 12-week period, and a list of runs with details such as status, duration, owner/participants and tasks finished.
```

Comment:
The final comma in this sentence is called the Oxford Comma or Serial Comma, and it has been included in this sentence intentionally.